diode-server/reconciler/changeset/changeset_virt_test.go

```diff
@@ -1138,12 +1138,7 @@ func TestVirtualizationPrepare(t *testing.T) {
 					queryParams:    map[string]string{"q": "undefined"},
 					objectChangeID: 0,
 					object: &netbox.DcimSiteDataWrapper{
-						Site: &netbox.DcimSite{
-							ID:     1,
-							Name:   "undefined",
-							Slug:   "undefined",
-							Status: (*netbox.DcimSiteStatus)(strPtr(string(netbox.DcimSiteStatusActive))),
-						},
```

Comment:
Could we cover both use cases: with existing site and without, please?

Reply:
Other than that, we should be good to go 🚀

Reply:
done @mfiedorowicz  see last commit